src/asm/stages/stage_3.asm

```diff
@@ -0,0 +1,199 @@
+.section .boot, "awx"
+.intel_syntax noprefix
+.code32
+
+
+# This stage checks for 64 bit, Sets up paging and enters long mode
+
+stage_3:
+	mov ax, 0x10
+	mov ss, ax
+    mov ds, ax
+    mov es, ax
+
+	call check_cpuid_support
+	call check_long_mode_support
+
+	cli # disable interrupts
+
+	lidt [zero_idt]
+
+	call setup_pae_paging
+	call enable_paging
+
+	lea si, paging_enabled_str
+	call print_string_32
+
+	lgdt gdt_64_pointer
+	push 8
```

Comment:
Before you pushed 0x8, try to be consistent with the counting bases so it's clear that it is for the same purpose

Reply:
fixed